.gitignore

```diff
@@ -65,3 +65,9 @@ python/kserve/tox.ini
 
 # Downloaded by hack install
 hack/istio-*
+
+# iTerm config
+.itermconfig
+
+# Docker config
+docker-compose.yml
```

Comment:
I think the docker-compose file is still useful, the one added was just in the wrong location.

Reply:
Where should we put it?